core/conn/jdbc_type2/native/CSrvrStmt.cpp

```diff
@@ -99,7 +99,7 @@ SRVR_STMT_HDL::SRVR_STMT_HDL(long inDialogueId)
     moduleName[0] = '\0';
     inputDescName[0] = '\0';
     outputDescName[0] = '\0';
-    isClosed = TRUE;
+    isClosed = FALSE;
```

Comment:
Changing the default value for isClosed from TRUE to FALSE can have other repercussions. We might miss throwing error. Can you please confirm this change.